rfq/negotiator.go

```diff
@@ -157,16 +157,24 @@ func (n *Negotiator) HandleOutgoingBuyOrder(buyOrder BuyOrder) error {
 	go func() {
 		defer n.Wg.Done()
 
-		// Query the price oracle for a bid price.
-		bidPrice, _, err := n.queryBidFromPriceOracle(
-			*buyOrder.Peer, buyOrder.AssetID,
-			buyOrder.AssetGroupKey, buyOrder.MinAssetAmount,
-		)
-		if err != nil {
-			err := fmt.Errorf("negotiator failed to handle price "+
-				"oracle response: %w", err)
-			n.cfg.ErrChan <- err
-			return
+		// We calculate a proposed bid price for our peer's
+		// consideration. If a price oracle is not specified we will
+		// skip this step.
+		var bidPrice lnwire.MilliSatoshi
+
+		if n.cfg.PriceOracle != nil {
+			// Query the price oracle for a bid price.
+			var err error
+			bidPrice, _, err = n.queryBidFromPriceOracle(
+				*buyOrder.Peer, buyOrder.AssetID,
+				buyOrder.AssetGroupKey, buyOrder.MinAssetAmount,
+			)
+			if err != nil {
+				err := fmt.Errorf("negotiator failed to "+
+					"handle price oracle response: %w", err)
+				n.cfg.ErrChan <- err
```

Comment:
Not sure if it's worth erroring out here? Or just log and continue without a proposed bid price?